### PR TITLE
filter tags lesser than one

### PR DESCRIPTION
**Pull request by AVRahul**
Signed-off-by: AVRahul <a.v.rahul11@gmail.com>

Filter tags if their consisting blog count is less than one.

Screenshot from 2021-07-13 14-49-35

### CODE REVIEW

**Comment by rakeshPRaghu** [comment]
Can we add the tags section as a separate component @AVRahul ? I think that will be better.
cc: @satyapriyamishra222

File: website/src/components/MiniBlog/index.tsx [hunk 1]
```diff
@@ -121,13 +121,20 @@ const MiniBlog: React.FC = () => {
...
   }, [jsonMdData]);
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => 
-          <StyledTab
-            label={`${item}(${tagsDistribution[item as keyof typeof tagsDistribution]})`}
-            value={item}
-            key = {item}
-          />
-  );
+  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
```
Comment (satyapriyamishra222) [inline website/src/components/MiniBlog/index.tsx:124]: Where is the trimming part that we discussed. Am I missing something ?

Comment (AVRahul) [reply website/src/components/MiniBlog/index.tsx:124]: Trim functionality is already available for the tags

Comment (AVRahul) [reply website/src/components/MiniBlog/index.tsx:124]: code at getPost.js for trim tags
``` 
                if (metadata){
                    post = {
                        id: index + 1,
                        title: metadata.title || 'No title',
                        author: metadata.author || 'No author',
                        author_info: metadata.author_info || 'No author information',
                        date: metadata.date || 'No date available',
                        **tags: metadata.tags.trim().split(',') || 'No tags available',**
                        excerpt: metadata.excerpt || '',
                        content: content || 'No content available'
                    };
                }
```

**Comment by AVRahul** [comment]
> Can we add the tags section as a separate component @AVRahul? I think that will be better.

No, @rakeshPRaghu  separate component is not required because at both the places the tag component is different at home and at the blog route.

File: website/src/pages/Blog/index.tsx [hunk 2]
```diff
@@ -161,13 +161,20 @@ const Blog: React.FC = () => {
...
 
-  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => 
-          <StyledTab
-            label={`${item}(${tagsDistribution[item as keyof typeof tagsDistribution]})`}
-            value={item}
-            key = {item}
-          />
-  );
+  const getTagsMarkup = Object.keys(tagsDistribution).map((item: string) => {
+    if (tagsDistribution[item as keyof typeof tagsDistribution] > 1) {
```
Comment (Pallavi-PH) [inline website/src/pages/Blog/index.tsx:165]: Should this filter be applied only for mobile or for all devices @satyapriyamishra222?

Comment (AVRahul) [reply website/src/pages/Blog/index.tsx:165]: IMO, All the devices 
cc @satyapriyamishra222

File: website/src/components/MiniBlog/index.tsx (hunk 1 above)
Comment (IsAmrish) [reply website/src/components/MiniBlog/index.tsx:131]: @AVRahul  -- aren't we removing the individual tags?

For eg.

```
`tag1,    tag2,    tag3.  , tag4`
```

**Review by IsAmrish — APPROVED** [review]

**Review by rakeshPRaghu — APPROVED** [review]

File: website/src/pages/Blog/index.tsx (hunk 2 above)
Comment (satyapriyamishra222) [reply website/src/pages/Blog/index.tsx:165]: Yes, let's use it across all devices. As we see it, mostly one timer tags are cluttering the space, let's filter them out for good.

**Review by Pallavi-PH — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]

**Pull request merged**